packages/core/src/repositories/documentVersionsRepository/index.ts

```diff
@@ -51,8 +63,8 @@ export class DocumentVersionsRepository extends Repository {
       .from(this.scope)
       .where(
         and(
-          eq(documentVersions.commitId, commit.id),
-          eq(documentVersions.documentUuid, documentUuid),
+          eq(this.scope.commitId, commit.id),
+          eq(this.scope.documentUuid, documentUuid),
```

Comment:
Now that we use scopes we always have to use the scope for picking the columns of the table. Otherwise it doesn't work

<img width="972" alt="image" src="https://github.com/user-attachments/assets/981512aa-3f6e-4888-a8f2-0f22e9f06dd8">

Reply:
💯 good catch